awpy/plot/plot.py

```diff
@@ -86,16 +81,14 @@ def plot(  # noqa: PLR0915
 
         # Plot each point
         for (x, y, z), settings in zip(points, point_settings):
-            transformed_x = position_transform_axis(map_name, x, "x")
-            transformed_y = position_transform_axis(map_name, y, "y")
+            transformed_x = game_to_pixel_axis(map_name, x, "x")
+            transformed_y = game_to_pixel_axis(map_name, y, "y")
             
-            if (ignore_extreme_points
-                and (
-                    transformed_x < 0
-                    or transformed_x > 1024
-                    or transformed_y < 0
-                    or transformed_y > 1024
-                )
+            if (
+                transformed_x < 0
+                or transformed_x > 1024
```

Comment:
Because these are constants, it would be helpful to have a comment above here stating that these are the maximum expected bounds